### ISSUE TITLE
Is this still being worked on?

### ISSUE THREAD

**Issue by UnsealedWings** [sd805/l4d2vr#47]
Hi! Love this mod SO MUCH, Would love to know if it's still being worked on and what is currently being done while we wait for updates?

**Comment by sd805** [comment]
Yeah, it's still being worked on. I haven't had much time to work on it recently so updates will be slower. I want to add roomscale movement next.
